index.php

```diff
@@ -6,6 +6,15 @@
 <main id="main-content">
 
 <?php
+// Exclude Sponsored posts, unless is search
+if( !is_search() ) {
+  // Get Sponsored cat ID
+  $sponsored_cat = get_category_by_slug('sponsored');
+  $sponsored_id = $sponsored_cat->cat_ID;
+
+  query_posts($query_string . '&category__not_in=' . $sponsored_id);
```

Comment:
this rly shud be in pre_get_posts https://codex.wordpress.org/Plugin_API/Action_Reference/pre_get_posts